google/cloud/pubsublite/google_cloud_cpp_pubsublite.bzl

```diff
@@ -35,6 +35,7 @@ google_cloud_cpp_pubsublite_hdrs = [
     "internal/alarm_registry.h",
     "internal/alarm_registry_impl.h",
     "internal/batching_options.h",
+    "internal/client_id_generator.h",
```

Comment:
The source of truth for these `.bzl` files is CMake.  You need to add these here:

https://github.com/googleapis/google-cloud-cpp/blob/e11ad2c3a59eeff9ddb9f435fe71b4f89672a9a4/google/cloud/pubsublite/CMakeLists.txt#L47

You may want to read

https://github.com/googleapis/google-cloud-cpp/blob/main/doc/contributor/working-with-bazel-and-cmake.md
https://github.com/googleapis/google-cloud-cpp/blob/main/doc/contributor/howto-guide-running-ci-builds-locally.md